.lycheeignore

```diff
@@ -1 +1,2 @@
-https://packagemanager.rstudio.com/cran/__linux__/focal/latest
\ No newline at end of file
+https://packagemanager.rstudio.com/cran/__linux__/focal/latest
+https://github.com/orgs/RConsortium/projects/7
```

Comment:
what does a .lycheeignore do?

Reply:
It validates that all links are working and accessible.  Since the project is private it can't validate it and throws an error.  We can add into the ignore so it no longer checks